index.html

```diff
@@ -12,16 +12,22 @@
 <body>
 <section class="topsection">
   <div class="container topsection__container">
-    <div class="topsection__header">
-      <div class="topsection__logo"></div>
-      <div class="topsection__header__items">
```

Comment:
BEM naming conventions usually forbid definition of element's element.

There are good articles that help to understand the ways of use and benefits of the BEM methodology:
http://getbem.com/naming/
https://en.bem.info/methodology/